modules/videoio/src/cap_msmf.cpp

```diff
@@ -1218,13 +1218,13 @@ double CvCapture_MSMF::getProperty( int property_id ) const
         return aspectN;
     else if (property_id == CV_CAP_PROP_SAR_DEN)
         return aspectD;
+    else if (property_id == CV_CAP_PROP_FRAME_WIDTH)
+        return isOpen ? captureFormat.width : requestedWidth;
```

Comment:
> isOpen ?

How we can create `cv::VideoCapture` where `isOpen == false`? In this case VideoCapture should trying the next backend from the list of enabled/available, and lost "this" instance.

How should we write MSMF test to test both branches of this condition?